### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by LeahWinters**
### Type of change made:
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Migration 
- [ ] Styling
- [X] Testing

### Commit Emojis:

:bar_chart: new tests

### Detailed Description:

Added in a couple of tests in the app component to:

- Ensure the header, weather, and destination components render on page load
- Ensure the user can select a destination and is brought to the voyage planning page

### Why is this change required? What problem does it solve?

Ensures the code base is working as expected

### Were there any challenges that arose while implementing this feature? If so, how were the addressed?

Tried to also pass a test to ensure the user can click the "learn more" button in the weather component. Currently having issues the our mock function not firing in the test. I believe it is because the function lives in the weather component instead of being passed as a prop from the app component. Will come back to this test. 

### Files modified:
- [X] App.test.js

### CODE REVIEW

**Review by lilianaweimer — APPROVED** [review]
These tests look great!

File: src/Components/App/App.test.js [hunk 1]
```diff
@@ -1,7 +1,123 @@
...
+
+    expect(header).toBeInTheDocument();
+    expect(newsTitle).toBeInTheDocument();
+    expect(newsDate).toBeInTheDocument();
+    expect(destinationName).toBeInTheDocument();
+
+  });
+
+  it.skip("Should be able to click on the learn more button and be redirected to a new page showing the article", async () => {
+    getRecentNews.mockResolvedValue(mockGetRecentNews);
```
Comment (lilianaweimer) [inline src/Components/App/App.test.js:80]: Really not sure why this one is failing. We should do some more research into `jest.mock()`

Comment (infamouskeyduster) [reply src/Components/App/App.test.js:80]: Good call @lilianaweimer. Much like this entire project, if we put our heads together, we can figure out anything 🧠 😄 !

**Review by infamouskeyduster — APPROVED** [review]
Good start to unit testing for App.js!

**Pull request merged**